gst/nnstreamer/nnstreamer_subplugin.c

```diff
@@ -29,6 +29,12 @@
 #include "nnstreamer_subplugin.h"
 #include "nnstreamer_conf.h"
 
+/** @brief Array of dlopen-ed handles */
+static GArray *handles = NULL;
+
+static void init_subplugin (void) __attribute__ ((constructor));
```

Comment:
how about adding priority value?
```
static void init_subplugin (void) __attribute__ ((constructor (101)));
static void fini_subplugin (void) __attribute__ ((destructor (101)));
```

Reply:
Do you have suggestions on the rules of priority values?

Reply:
plz ignore this comment